Notes

```diff
@@ -1,9 +1,9 @@
 # To build:
-autoreconf -i
-CC="gcc-4.8.3" CFLAGS="-fms-extensions -std=c11" ./configure && make
-# You shouldn't need that exact version of gcc, it just has to be recent and
-# you shouldn't really need the -fms-extensions either if yours is up to date
-# enough.
+mkdir build
+meson . build
```

Comment:
I just did `meson build` according to the instructions on http://mesonbuild.com/Quick-guide.html when playing with [rage](https://github.com/concert/rage). Which is preferred and why?

Reply:
That'd be fine, I think what I was doing might be an old way.